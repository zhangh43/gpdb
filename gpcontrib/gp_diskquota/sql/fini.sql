\! gpconfig -c diskquota.monitor_databases -v postgres > /dev/null
\! echo $?
-- start_ignore
\! gpstop -u > /dev/null
\! echo $?
-- end_ignore

\! sleep 2
