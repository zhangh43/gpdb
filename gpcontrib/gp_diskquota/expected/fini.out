\! gpconfig -c diskquota.monitor_databases -v postgres > /dev/null
\! echo $?
0
-- start_ignore
\! gpstop -u > /dev/null
\! echo $?
0
-- end_ignore
\! sleep 2
